examples/README.md

```diff
@@ -1,4 +1,4 @@
 # Python Functions Frameworks Examples
 
 * [`cloud_run_http`](./cloud_run_http/) - Deploying an HTTP function to [Cloud Run](http://cloud.google.com/run) with the Functions Framework
-* [`cloud_run_event`](./cloud_run_event/) - Deploying a CloudEvent function to [Cloud Run](http://cloud.google.com/run) with the Functions Framework
+* [`cloudevents`](./cloudevents/) - Deploying a CloudEvent function to [Cloud Run](http://cloud.google.com/run) with the Functions Framework
```

Comment:
```suggestion
* [`cloud_run_cloudevents`](./cloud_run_cloudevents/) - Deploying a CloudEvent function to [Cloud Run](http://cloud.google.com/run) with the Functions Framework
```

Since the example is specifically about Cloud Run. This also mirrors the HTTP example.